test/integration/metrics_test.go

```diff
@@ -340,6 +311,30 @@ func TestAllMetricsPresent(t *testing.T) {
 	})
 }
 
+// fetcher is a helper function that returns a Fetcher.
+//
+// If any error happens, the function will make to fail the given test.
+func fetcher(t *testing.T, docker *client.Client) raw.Fetcher {
+	t.Helper()
+
+	cgroupInfo, err := raw.GetCgroupInfo(context.Background(), docker)
+	if err != nil {
+		t.Fatalf("Creating CgroupInfo: %v", err)
+	}
```

Comment:
Nit: since testify is already used in the project, you may want to use it to check errors (it is a bit easier to read).

```go
require.NoError(t, err)
```

Reply:
You're right, updated!